backend/src/args.rs

```diff
@@ -146,10 +146,10 @@ fn parse_color_choice(s: &str) -> Result<ColorChoice, &'static str> {
 
 impl Args {
     pub(crate) fn stdout_color(&self) -> ColorChoice {
-        if std::io::stdout().is_terminal() { self.color } else { ColorChoice::Never }
+        self.color
     }
 
     pub(crate) fn stderr_color(&self) -> ColorChoice {
-        if std::io::stderr().is_terminal() { self.color } else { ColorChoice::Never }
+        self.color
```

Comment:
To be fair, I didn't change what this does in https://github.com/elan-ev/tobira/commit/acac4afb351614ce5491d545ae3558048f5117b3, so I can't tell you why.